### PR TITLE
Getting Started in Contribution.md

### PR DESCRIPTION
**Pull request by i-am-SnehaChauhan**
Issue #11 
I have completed the " getting started" section in the contribution.md Please review it. Thank you

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -31,14 +31,65 @@ If you're new to open source and don't know where to start, don't worry! We've t
...
+```
+git remote -v
+```
+### Step 5. Keep Your Local Copy Updated
+```
+git pull upstream main
+``` 
+### Step 6. Create a New Branch
+
+```git checkout -b <your_branch_name>
```
Comment (ajaynegi45) [inline contributing.md:63]: This line is not visible in preview

Screenshot 2023-10-06 at 8 28 46 PM

**Review by ajaynegi45 — CHANGES_REQUESTED** [review]

**Comment by i-am-SnehaChauhan** [comment]
Hey @ajaynegi45, why did you close this pull request without merging it into the main branch

**Comment by i-am-SnehaChauhan** [comment]
I have made changes. Can you please check it again?

**Comment by ajaynegi45** [comment]
> I have made changes. Can you please check it again?

Read this issue https://github.com/ajaynegi45/Library/pull/29#discussion_r1348842353
after resolving, create new pr

**Comment by i-am-SnehaChauhan** [comment]
Done

**Pull request closed without merging**